scripts/copy-files.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+set -eu
+
+# Palette needs to be accessible from stylus and from JS
+JS_PALETTE='react/palette.js'
```

Comment:
I was expecting this to be `transpiled/react/palette.js` but it seems to work, not sure why (I'm probably missing something).

Reply:
I want it to be both in source `react` and transpiled `react`.